draft-aegis-mimi-arch-latest.md

```diff
@@ -311,13 +379,13 @@ HTTP PATCH /apps/{appId}/clients/{clientId}
 Input:
 
 {
-   clientIdentity: Blob,
+   ClientID: Blob,
```

Comment:
Again, should be `clientIdentity` to update, e.g. alice_smith becomes alice_johnson